plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/MetadataEntry.java

```diff
@@ -120,7 +120,7 @@ public List<String> getOriginalPartitionColumns()
      * For use in read-path. Returns lowercase partition column names.
      */
     @JsonIgnore
-    public List<String> getCanonicalPartitionColumns()
+    public List<String> getLowercasePartitionColumns()
```

Comment:
IDE marks it as unused

Reply:
This method is used by non-test methods as well. 

<img width="1013" alt="Screenshot 2023-08-07 at 15 27 01" src="https://github.com/trinodb/trino/assets/6237050/c8a9b51d-9d95-42aa-8058-65b26e1890e2">